HELM-NOTES.md

```diff
@@ -0,0 +1,196 @@
+# Deploying Apache Hop with Helm on Kubernetes
+
+This document contains technical documentation on how to deploy Apache Hop using Helm on a Kubernetes cluster.
+
+## What is Helm?
+
+Kubernetes can become very complex to manage using standard configuration files (services, pods, volumes, different releases, ... ) which is solved by the use of Helm. Helm is a package manager which can package everything into a simple configurable application. Helm offers easy installing, updating and removal.
+
+> Helm installs charts into Kubernetes, creating a new release for each installation. And to find new charts, you can search Helm chart repositories.
+
+## Prerequisites
+
+The following prerequisites are needed to get started with Apache Hop and Helm.
+
+- A repository which contains a working Hop Docker image
+- A kubernetes cluster (Amazon EKS, Google GKE, minikube, ... )
+- Helm [installed and configured](https://helm.sh/docs/intro/install/)
+
+## The Hop Helm Chart
+
+The following files form the Hop Helm Chart.
+
+```
+hop/
```

Comment:
I would recommend have directory structure as below
hop-docker/
    kubernetes/
        helm/
            Chart.yaml
            charts/
            templates/
            values.yaml
            README.md

Reply:
Updated the directory structure for Helm, maybe the k8s directory should be moved out of examples too?